src/lualib/5.0/Error.ts

```diff
@@ -0,0 +1,71 @@
+interface ErrorType {
```

Comment:
Not a huge fan of the code duplication. As far as I can tell this is almost identical with the existing Error.ts, except (wrapErrorToString). Do you think it would be possible to use the existing Error.ts and work around the difference by using `_VERSION`? I think it's fine  to add an additional condition since errors are not performance critical.

Reply:
Yes, good idea! Implemented with https://github.com/TypeScriptToLua/TypeScriptToLua/pull/1263/commits/5549a1c83e6374b1831e3dec77895c8e2b953769.